### ISSUE TITLE
how can i acheive screen share

### ISSUE THREAD

**Issue by Boopalan-M** [ChandanRabha/adbu_video_chat#1]

**Comment by ChandanRabha** [comment]
You can use PeerJS

**Comment by Boopalan-M** [comment]
yes i use that but can't able to achieve that.. the screen is visible to us
onlyy

**Comment by ChandanRabha** [comment]
https://www.youtube.com/watch?v=X8QHHB7DA90&t=2s
You can watch this video, basically you have to change the video Mediastream of the webcam you are sending to the peers and change it to screen Mediastream.
